frontend/components/panels/WalletVatDaiBalanceCheckPanel.vue

```diff
@@ -100,6 +103,12 @@ export default Vue.extend({
                     title: 'The VAT balance is unknown until a wallet is connected',
                 };
             }
+            if (this.transactionBidAmount.isNaN()) {
+                return {
+                    name: 'incorrect',
```

Comment:
As suggested above, let's make it `inactive`, not `incorrect`

Reply:
Oh sorry, misread that 😅